rhizome/host/lib/vm_setup.rb

```diff
@@ -440,10 +441,18 @@ def cloudinit(unix_user, public_keys, gua, nics, swap_size_bytes, boot_image, dn
     runner_config = if boot_image.include?("github")
       <<~ADDRESSES
       address=/ubicloudhostplaceholder.blob.core.windows.net/#{nics.first.net4.split("/").first}
-      filter-AAAA
+      server=9.9.9.11
+      server=149.112.112.11
+      server=2620:fe::11
+      server=2620:fe::fe:11
```

Comment:
very astute. I remember @enescakir was mentioning EDNS and I waffled on it, but if this casually helps with CDNs that serve a bunch of code, I think that's a lock: we should enable it. The privacy considerations are not the same for end users versus servers, I don't think.

Reply:
https://en.wikipedia.org/wiki/Quad9

<img width="944" alt="Screenshot 2024-11-26 at 22 28 37" src="https://github.com/user-attachments/assets/1b29c3c1-2735-4e15-adad-9da0ebe50266">

Reply:
<img width="392" alt="Screenshot 2024-11-26 at 22 29 43" src="https://github.com/user-attachments/assets/f03ca18b-418e-44b7-89cc-9b30d5cb346d">

Reply:
Turned out that was not necessary, it was all about the docker + dns